.github/actions/waffles/README.md

```diff
@@ -44,8 +44,8 @@ None other than the log output and system exit code.
 
 uses: actions/waffles@v1
 with:
-  app-libs: '/Projects/cds/notification-document-download-api'
   app-loc: '/doc-api-env/Lib/site-packages'
+  app-libs: '/Projects/cds/notification-document-download-api'
```

Comment:
`app-loc` should be '/Projects/cds/notification-document-download-api' and `app-libs` '/doc-api-env/Lib/site-packages', right?

Reply:
Right, I changed that.